src/components/CookieWarning.js

```diff
@@ -13,17 +14,55 @@ import { authStore } from 'src/libs/state'
 export const cookiesAcceptedKey = 'cookiesAccepted'
 
 const CookieWarning = () => {
+  const animTime = 0.3
   const signal = useCancellation()
+  const [showWarning, setShowWarning] = useState(false)
+  const [opacity, setOpacity] = useState(0)
   const { cookiesAccepted } = useStore(authStore)
+  const timeout = useRef()
 
   const acceptCookies = acceptedCookies => {
     authStore.update(_.set('cookiesAccepted', acceptedCookies))
   }
 
+  useEffect(() => {
+    if (cookiesAccepted === undefined) {
+      timeout.current = setTimeout(showComponent, 3000)
```

Comment:
I personally would check with Sarah Tahiri to see what she thinks about it from a compliance perspective. I'm worried that is too long still.

Reply:
Yeah, it could be! I basically decided on this number because it just "felt good" to me. Have you tried it in the deployed pr branch?

https://pr-9-dot-bvdp-saturn-dev.appspot.com/